music/demos/music_rnn.ts

```diff
@@ -144,9 +148,13 @@ function writeNoteSeqs(elementId: string, seqs: mm.INoteSequence[]) {
 }
 
 function createPlayer(seq: mm.INoteSequence) {
-  const player = new mm.Player();
+  const player = new mm.GMPlayer(SOUNDFONT_URL);
   const button = document.createElement('button');
   button.textContent = 'Play';
+  button.disabled = true;
+  player.initialize()
+      .then(() => player.loadSamples(seq))
```

Comment:
shouldn't this be called inside "start" so the user doesn't have to remember to call it each time?

Reply:
Similar to above, my intention was to more closely match the behavior of the Player class, where "start" schedules everything immediately.

I could add another method loadSamplesAndStart...

Reply:
I think this is pretty confusing since you have to call 2 methods every time you want to play something.  I'd argue you should either just lazily load the samples when you call start(), or load all of the samples in initialize().